pages/1.13/security/oss/login/index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: layout.pug
+navigationTitle: Login
+title: Login
+excerpt: Logging in to your DC/OS cluster
+menuWeight: 20
+
+---
+
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+# User login
+
+With DC/OS, login is the process of exchanging user credentials for a [DC/OS Authentication token](/1.13/security/oss/authentication/authentication-token/).
+
+Obtaining a DC/OS Authentication is required by any user before being able to use a DC/OS cluster. In Open DC/OS the life-time of an Authentication token is limited to five days. Once the Authentication token expires the user must log in again.
```

Comment:
I am actually not sure if "Open DC/OS" is what we should write here. Should we just say "DC/OS"? Please check back with the documentation team. This applies to the entire patch.